includes/classes/main/class-layouts-single.php

```diff
@@ -30,6 +30,7 @@ public function current_screen() {
 
             // Single
             add_action( 'load-post.php', array( $this, 'load_single' ) );
+            add_action( 'load-post.php', array( $this, 'has_screenshot_file' ) );
```

Comment:
@michael-dev-pilotin I can't find any function with this name. Forgot to remove this line?